piksi_tools/console/settings_view.py

```diff
@@ -287,7 +287,9 @@ def set(self, section, name, value):
       self.link.send(SBP_MSG_SETTINGS,
           '%s\0%s\0%s\0' % (section, name, value))
 
-  def __init__(self, link, read_finished_functions=[], gui_mode=True):
+  def __init__(self, link, read_finished_functions=[],
+               name_of_yaml_file="settings.yaml", gui_mode=True):
```

Comment:
I may add an arg to the console.py to maybe specify the path to be used in the constructor